### ISSUE TITLE
Add index on `chunks` table for `block_hash`

### ISSUE THREAD

**Issue by frol** [near/near-indexer-for-explorer#36]
Confirm that the following query is immediate (it currently takes around 2-4 minutes):

```sql
select * from chunks where block_hash = '51DkcKsnEfPgBwuBCbNTTEfHDR6P8dXWN9r2PfYBXnRE' limit 1;
```

**Comment by bowenwang1996** [comment]
> it currently takes around 2-4 minutes

Why?

**Comment by frol** [comment]
Because of the missing index. We need to add the index on the column to fix it.

**Comment by khorolets** [comment]
Index added and applied for mainnet and testnet databases. I confirm your query is taking millis to execute, though I don't know how to measure it exactly.

Migration included in PR #39

**Comment by khorolets** [comment]
Closed in #39
